### PR TITLE
Bugfix in fixImage.sh

### PR DESCRIPTION
**Pull request by AndrewFasano**
Simple bugfix, happy to explain more if necessary. Fixes the issue I described in https://github.com/firmadyne/firmadyne/pull/162#issuecomment-720771869

### CODE REVIEW

**Comment by ddcc** [comment]
Thanks for point this out. Shouldn't the correct fix be to prepend `$BUSYBOX ` to all of the commands; e.g. cp, mv, rm, mknod, and mkdir?

**Comment by brianpow** [comment]
Would you check if your host busybox has 'mv' built-in ?

**Comment by ddcc** [comment]
All of those commands are included in busybox for me, including mv.

**Comment by AndrewFasano** [comment]
Just updated this to use `$BUSYBOX mv` instead. Also changed the other commands to be prefixed with `$BUSYBOX` which is probably safer even though they all seemed to be working without it.

File: scripts/fixImage.sh [hunk 1]
```diff
@@ -15,50 +15,50 @@ resolve_link() {
...
+    $BUSYBOX mkdir -p "$(dirname $(resolve_link /etc/hosts))"
     echo "127.0.0.1 localhost" > "$(resolve_link /etc/hosts)"
 fi
 
 PASSWD=$(resolve_link /etc/passwd)
 SHADOW=$(resolve_link /etc/shadow)
 if [ ! -s "$PASSWD" ]; then
     echo "Creating $PASSWD!"
-    mkdir -p "$(dirname $PASSWD)"
+    $BUSYBOX $BUSYBOX mkdir -p "$(dirname $PASSWD)"
```
Comment (0BuRner) [inline scripts/fixImage.sh:53]: Mistake?

**Pull request merged**